plugins/org.locationtech.udig.libs/META-INF/MANIFEST.MF

```diff
@@ -13,8 +13,7 @@ Require-Bundle: org.eclipse.core.runtime;visibility:=reexport,
  org.apache.xerces;visibility:=reexport,
  org.apache.log4j;visibility:=reexport,
  org.eclipse.xsd;visibility:=reexport,
- org.eclipse.swt;visibility:=reexport,
- org.apache.commons.codec;bundle-version="1.9.0";visibility:=reexport

```

Comment:
good idea not tu use required bundle. others might have a never version of from another provider. This makes live easier from integration point of view :)